turbinia/pubsub.py

```diff
@@ -60,20 +60,22 @@ def setup(self):
   def setup_publisher(self):
     """Set up the pubsub publisher."""
     config.LoadConfig()
-    self.publisher = pubsub.PublisherClient()
+    self.publisher = gcp_common.CreateService('pubsub', 'v1')
     self.topic_path = self.publisher.topic_path(
         config.TURBINIA_PROJECT, self.topic_name)
     try:
       log.debug('Trying to create pubsub topic {0:s}'.format(self.topic_path))
-      self.publisher.create_topic(self.topic_path)
+      topics_client = self.publisher.topics()
+      gcp_common.ExecuteRequest(
+          topics_client, 'create', {'name': self.topic_path})
     except exceptions.Conflict:
       log.debug('PubSub topic {0:s} already exists.'.format(self.topic_path))
     log.debug('Setup PubSub publisher at {0:s}'.format(self.topic_path))
 
   def setup_subscriber(self):
     """Set up the pubsub subscriber."""
     config.LoadConfig()
-    self.subscriber = pubsub.SubscriberClient()
+    #self.subscriber = gcp_common.CreateService('pubsub', 'v1')
```

Comment:
```suggestion
```

Reply:
maybe leave this to use it below and do the `self.subscriber.subscribe(subscription_path, self._callback)` call in case it is used by the server not by the client code